drupal/sites/default/boinc/themes/boinc/templates/views-view-table--boinc-team-forum-topics.tpl.php

```diff
@@ -51,29 +50,29 @@
       if ($topic_index == 1) {
         $row_class .= ' first';
       }
-      if ($topic->sticky=="True") {
+      if ($result[$id]->node_boincteam_forum_node_sticky) {
         $row_class .= ' sticky';
       }
-      elseif (!$first_non_sticky AND !($topic->sticky=="True")) {
+      elseif (!$first_non_sticky AND !($result[$id]->node_boincteam_forum_node_sticky)) {
         $row_class .= ' first-non-sticky';
         $first_non_sticky = TRUE;
       }
       if ($topic_index == $topic_count) {
         $row_class .= ' last';
       }
-      if ($topic->timestamp=="new" OR $topic->new_comments) {
+      if ($topic->timestamp OR $topic->new_comments) {
```

Comment:
It's a hacky solution to the problem, `$topic->timestamp` is a string which is either blank (empty) or ==`new`. The problem is that `new` is translated into a language, example `neu`, by drupal. Therefore a string comparison won't work.

Thus my solution is to simply check the existence of `$topic->timestamp`. If the topic is not new, then the field should be empty.